INSTALL.md

```diff
@@ -12,22 +12,16 @@ Your node.js ships npm in a suitable version.
 
 Some modules are compiled during the installation. Therefore some additional software must be installed.
 
-### Python
+### Python & C++
 
-* Python 2.7 (not Python 3!) in system path
-* For Mac OS: Python 2.7 is already installed.
-* For Windows:
-  * Make sure that Python and node.js are both either 32 bit or 64 bit - mixed combinations will not work!
-  * Add an environment variable `PYTHON` that points to the Python executable.
+In former versions of this project it was definitely neccessary to have Python and a C++ compiler installed on your local dev machine.
+This was because we indirectly used modules that itself had native code inside that needs to be compiled. For these native 
+modules nodejs uses `node-gyp`. Please have a look at their github project for further information. 
+https://github.com/nodejs/node-gyp
```

Comment:
I want to point the reader to potentially helpful information if she runs into problems because some dependency is still or again using native libs